### ISSUE TITLE
Implement authorization

### ISSUE THREAD

**Issue by bpross-52n** [52North/javaPS#79]
Certain endpoints/operations need a security layer, e.g. getting the list of jobs, executing processes and deploying new processes.

**Comment by Kundanagrawalofficial** [comment]
Yes, that is correct. To ensure the security of sensitive information and to control access to certain resources, many APIs require authentication and authorization. An authentication token, such as an OAuth token or JSON Web Token (JWT), is typically used to identify the user or client making the API request. Once the user is authenticated, an authorization check is performed to determine if the user has the necessary permissions to access the requested endpoint or operation.

If the user is not authorized, the API returns a "Not Authorized" error, indicating that the user lacks the necessary permissions to access the requested resource. In this case, the user may need to request access or provide additional credentials to gain the necessary authorization.

**Comment by njiddasalifu** [comment]
Hello I wish to work on this issue.  some one should assign it to me

**Comment by ayaan474** [comment]
i would like to solve this issue . can u please assign it to me.

**Comment by AbhayXplorer** [comment]
Hello I’m interested in working on this issue as part of my open-source contribution.  
Could you please assign this issue to me?
